### PR TITLE
Depreciation in ReGex

### PR DESCRIPTION
**Pull request by ariG23498**
This PR:
- Removes the depreciation warning in the test.
- Uses raw string for the regex expression.

Should fix: #11

### CODE REVIEW

File: mydocstring/parse.py [hunk 1]
```diff
@@ -42,9 +42,9 @@ class DocString(object):
 
     Attributes:
         delimiter: A string that is used to identify a section and is placed
-            after the section name (e.g., `Arguments:`). Defaults to `': '`.
+            after the section name (e.g., `Arguments:`). Defaults to `':'`.
         arg_delimiter: A string that specifies how to separate arguments in a
-            argument list. Defaults to `':'`.
+            argument list. Defaults to `': '`.
```
Comment (ooreilly) [inline mydocstring/parse.py:47]: why remove a space on line 45 and add one on line 47? let's be consistent and do the same on both lines, no space.

File: mydocstring/parse.py [hunk 2]
```diff
@@ -695,6 +719,10 @@ def mark_code_blocks(txt, keyword='>>>', split='\n', tag="```", lang='python'):
 
     for block in blocks:
         lines = block.split(keyword)
+                           # (\s*)            - capture the spaces
+                           #      (%s         - start capture >>>lines
+                           #         [\w\W]+  - one or more characters and special characters
+                           #                ) - end capture
         match = re.findall(r'(\s*)(%s[\w\W]+)' % keyword,
                            keyword.join(lines[1:]), re.M)
         if match:
```
Comment (ooreilly) [inline mydocstring/parse.py:728]: nice with all the descriptions!

**Review by ooreilly — COMMENTED** [review]
It's great to see that the deprecation issues have been resolved. Thank you for taking care of that and also for documenting the regular expressions.

File: mydocstring/parse.py (hunk 1 above)
Comment (ariG23498) [reply mydocstring/parse.py:47]: I had to interchange the spaces due to these lines of code:
https://github.com/ooreilly/mydocstring/blob/0eaf978effa77e52ec3e8b6b17b235e00fb84016/mydocstring/parse.py#L91-#L92

Comment (ooreilly) [reply mydocstring/parse.py:47]: I see. Thanks for making the change so that the documentation is consistent with the code.

**Pull request merged**